Formula/autogen.rb

```diff
@@ -13,16 +12,12 @@ class Autogen < Formula
     sha256 "ff8c66ca7d86c309e884dad0fcc49aadf65a830768a0551c5711cba2f6d6a046" => :el_capitan
   end
 
+  depends_on "coreutils" => :build
   depends_on "pkg-config" => :build
   depends_on "guile"
 
-  # Allow guile 2.2 to be used
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/0de886b/autogen/allow-guile-2.2.diff"
-    sha256 "438fe673432c96d5da449b84daa4d1c6ad238ea0b4ccd13491872df8c51fa978"
-  end
-
   def install
+    inreplace %w[agen5/mk-stamps.sh build-aux/run-ag.sh config/mk-shdefs.in], "mktemp", "gmktemp"
```

Comment:
Thanks a lot for this! Can you include a link to the bug report above it? e.g.

```suggestion
    # Uses GNU-specific mktemp syntax: https://sourceforge.net/p/autogen/bugs/189/
    inreplace %w[agen5/mk-stamps.sh build-aux/run-ag.sh config/mk-shdefs.in], "mktemp", "gmktemp"
```